core/org.eclipse.cdt.core/parser/org/eclipse/cdt/internal/core/dom/parser/cpp/semantics/EvalPackAccess.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright (c) 2016,2022 Institute for Software, HSR Hochschule fuer Technik and others
+* Copyright (c) 2016,2022 Institute for Software, HSR Hochschule fuer Technik
```

Comment:
Thanks for fixing this up, I guess the next line should actually be deleted?

Reply:
No, I believe this should be kept as, two lines actually match attributions in header of `EvalCompositeAccess.java` except for new years of contribution.

Reply:
Yes, I see and understand now. Thanks.